packages/cli/DEVELOP.md

```diff
@@ -0,0 +1,27 @@
+# Developer Guide
+
+## Running Tests
+
+### All tests
+
+`yarn test`
+
+### Anvil & Ganache tests
+
+There are 2 more yarn commands:
+
+- `yarn test-anvil`
+- `yarn test-ganache`
+
+They will run anvil and ganache tests (+ common tests that are neither of those) ONLY by providing necessary process env variables. Those are:
```

Comment:
it knows which tests are anvil test because they have the `testWithAnvil` function. which internally checks the RUN_ANVIL_TESTS env var